ml/src/main/scala/org/apache/spark/ml/h2o/algos/H2OAutoML.scala

```diff
@@ -157,45 +154,20 @@ object H2OAutoML extends MLReadable[H2OAutoML] {
   override def load(path: String): H2OAutoML = super.load(path)
 }
 
-// FIXME: H2O Params are iced objects!
-private[algos] class H2OAutoMLWriter(instance: H2OAutoML) extends MLWriter {
-
-  @Since("1.6.0") override protected def saveImpl(path: String): Unit = {
-    val hadoopConf = sc.hadoopConfiguration
-    DefaultParamsWriter.saveMetadata(instance, path, sc)
-    val outputPath = new Path(path, instance.defaultFileName)
-    val fs = outputPath.getFileSystem(hadoopConf)
-    val qualifiedOutputPath = outputPath.makeQualified(fs.getUri, fs.getWorkingDirectory)
-    val out = fs.create(qualifiedOutputPath)
-    val oos = new ObjectOutputStream(out)
-    oos.writeObject(instance.automlBuildSpec.orNull)
```

Comment:
Same as above, we create automl build spec and set up as we need during `fit`, If we set up any arguments, the are stored by Spark and we just re-apply them in `fit`